amr-wind/setup/set_background_pressure.cpp

```diff
@@ -44,6 +44,7 @@ void incflo::set_background_pressure()
 
             if (delp_dir == -1) {
                 delp_dir = dir;
+                // cppcheck-suppress unreadVariable
```

Comment:
let's delete this entire function, we have bodyForce now

Reply:
It's not obvious to me how to do so. It's called here https://github.com/Exawind/amr-wind/blob/da8509fed40eb7508ae7b70f2b0f7996f161aef9/amr-wind/incflo.cpp#L32

Reply:
yep, just delete that too! I'm pretty sure we never enter the logic inside that function. And if we did (mix of pressure inflow and pressure outflow) it would abort